packages/@lightningjs/ui-components/src/components/FocusManager/FocusManager.mdx

```diff
@@ -119,35 +119,58 @@ class FancyFocus extends FocusManager {
-| selectedIndex | number                                                  | false    | index of current item in focus                                                                                               |
-| loading       | boolean                                                 | true     | False if an item is in focus, true otherwise                                                                                 |
-| items         | <DocsLink id="lng.Component">lng.Component[]</DocsLink> | false    | alias for `children`                                                                                                         |
-| wrapSelected  | boolean                                                 | false    | enables wrapping behavior, so `selectNext` selects the first item if the current item is the last on the list and vice versa |
+| name          | type            | default   | description                                                    |
+| ------------- | --------------- | --------- | -------------------------------------------------------------- |
+| direction     | 'column'\|'row' | undefined | The navigation direction for focus ('left/right' or 'up/down') |
+| selectedIndex | number          | undefined | The index of the currently selected item.                      |
+| Items         | FocusItemsType  | undefined | Child element or elements of the FocusManager.                 |
+| wrapSelected  | boolean         | false     | Enables wrapping behavior for focus navigation.                |
 
 ### Methods
 
-#### appendItems
+#### appendItems(items: <DocsLink id="lng.Component">lng.Component[]</DocsLink>): void
 
-Appends items to child list and updates focus
+This method appends the provided rows to the end of the FocusManager's children
+
+#### appendItemsAt(items: <DocsLink id="lng.Component">lng.Component[]</DocsLink>, idx: number): void
+
+This method is similar to appendItems, but it allows appending items at a specific index in the list of child items.
 
 ##### Arguments
 
-| name  | type                                                    | required | default | description     |
-| ----- | ------------------------------------------------------- | -------- | ------- | --------------- |
-| items | <DocsLink id="lng.Component">lng.Component[]</DocsLink> | false    | []      | items to append |
+| name  | type                                                    | required | default | description                   |
+| ----- | ------------------------------------------------------- | -------- | ------- | ----------------------------- |
+| items | <DocsLink id="lng.Component">lng.Component[]</DocsLink> | false    | []      | The list of items to append   |
+| idx   | number                                                  | true     | []      | The index to insert the items |
```

Comment:
The default for `idx` should be `undefined` since it's has a type of number rather than an empty array